mod_reforged/hooks/config/character_injuries.nut

```diff
@@ -0,0 +1,74 @@
+// Regular Damage Types
+::Const.Injury.BluntHead.extend([
```

Comment:
I'm sorry for missing this in my earlier reviews but I just noticed it now that these injuries should also additionally be added to the following vanilla arrays:
1. All three injuries go in `::Const.Injury.All`.
2. Black Eye and Dislocated Jaw go in `::Const.Injury.BluntAndPiercingHead`
3. Heat Stroke goes in `::Const.Injury.BurningAndPiercingBody` and `::Const.Injury.Burning`.